src/containers/ArticlePage/TopicArticlePage/components/TopicArticleConnections.tsx

```diff
@@ -0,0 +1,192 @@
+/**
+ * Copyright (c) 2017-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { Fragment, useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { Structure } from '@ndla/editor';
+import { FieldHeader } from '@ndla/forms';
+import { colors } from '@ndla/core';
+import Button from '@ndla/button';
+import { injectT, tType } from '@ndla/i18n';
+import Modal, { ModalHeader, ModalBody, ModalCloseButton } from '@ndla/modal';
+import { Switch } from '@ndla/switch';
+import { fetchUserData } from '../../../../modules/draft/draftApi';
+import { HowToHelper } from '../../../../components/HowTo';
+import StructureFunctionButtons from './StructureFunctionButtons';
+import ActiveTopicConnections from '../../../../components/Taxonomy/ActiveTopicConnections';
+import { Topic, SubjectType } from '../../../../interfaces';
+import { PathArray, Input } from '../../../../util/retriveBreadCrumbs';
+
+const StyledTitleModal = styled('h1')`
+  color: ${colors.text.primary};
+`;
+
+const ModalTitleRow = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+`;
+
+const TopicArticleConnections = (props: Props & tType) => {
```

Comment:
```suggestion
const TopicArticleConnections = ({ t, structure, activeTopics, ...rest}: Props & tType) => {

```
også kan du fjerne alle steder det står `porps._VARIABELNAVN_`

Reply:
aah, men da må du sikkert endre på speadingen på linje 135 til `{...rest}` 
har ikke kjørt det opp med alle endringene selv så har ikke testet at det blir rett.